types/datum.go

```diff
@@ -420,8 +424,8 @@ func (d *Datum) GetValue() interface{} {
 	}
 }
 
-// SetValue sets any kind of value.
-func (d *Datum) SetValue(val interface{}) {
+// SetValueForTest sets any kind of value.
+func (d *Datum) SetValueForTest(val interface{}) {
```

Comment:
What's the difference between SetValue and SetValueForTest?

Reply:
There're many tests called it without any collation information.

Reply:
There're many tests called it without any collation information.